### PR TITLE
(aggs): Add adjacency matrix aggregation

### PR DESCRIPTION
**Pull request by Sharptsa**
I added the AdjacencyMatrix aggregation

### CODE REVIEW

**Comment by Sharptsa** [comment]
Looks like there is an issue with the tests. It fails on my branch, but also on master on the same tests. 
Don't know what is going on.

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ All notable changes to this project will be documented in this file based on the
 * support for elasticsearch-php ^7.0
 * Added `ParentAggregation` [#1616](https://github.com/ruflin/Elastica/pull/1616)
 * Elastica\Reindex missing options (script, remote, wait_for_completion, scroll...)
+* Added `AdjacencyMatrix` aggregation
```
Comment (ruflin) [inline CHANGELOG.md]: Could you add the PR number here?

Comment (Sharptsa) [reply CHANGELOG.md]: Yes, PR #1639 fixed the tests.
I pushed the smal change. :wink:

**Comment by ruflin** [comment]
@Sharptsa Thanks for the contribution!

**Pull request merged**